index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+   <meta charset="UTF-8" />
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <title id="title"></title>
+   <link rel="stylesheet" href="styles.css" />
+   <script src="index.js"></script>
```

Comment:
Js script tag should be under body tag.

And you can search why it should be so